### ISSUE TITLE
Deduplicate should ignore parts in URL after hashmark (Fragment identifier)

### ISSUE THREAD

**Issue by mghomedev** [Mimieam/TabSorter2#10]
Please add a setting to ignore all parts of URL after the "hash mark" for the "Deduplicate" button.

See also https://en.wikipedia.org/wiki/Fragment_identifier

The main purpose of the feature request is to work with ticket systems like trac ( see https://trac.edgewall.org/ ) and redmine (see https://www.redmine.org/ ).

We often have the same ticket open multiple times, but the URL is slightly different due to the hash tags in the URL, e.g. in trac it is like this:

http://tracserver.com/ticket/11393#comment:2 and http://tracserver.com/ticket/11393#comment:5 which are both same ticket "11393" and should be de-duplicated.
Same issue with redmine, there we have the example

https://redmineserver.com/redmine/issues/71087#change-386359 and https://redmineserver.com/redmine/issues/71087#note-8 are actually the same ticket.
Essentially, the de-duplicate function should ignore everything after the hashtag function "#" . This could be configurable in the settings menu.
To be honest, I think most people would enable that setting.

**Comment by Mimieam** [comment]
That sounds like a great suggestion, I will be adding this to my todo list :)

**Comment by nstielau** [comment]
Agreed.  Google Docs does this quite a bit as well, based on different sections of the document or slides in the deck.

**Comment by shawn-yee** [comment]
Adding a custom whitelist option, may simplify the logic in the short term
